modules/API/external/external_api.js

```diff
@@ -93,10 +93,12 @@ function changeParticipantNumber(APIInstance, number) {
  * @param interfaceConfigOverwrite object containing configuration options
  * defined in interface_config.js to be overridden.
  * @param noSsl if the value is true https won't be used
+ * @param {string} [jwt] the JWT token if needed by jitsi-meet for
+ * authentication.
  * @constructor
  */
 function JitsiMeetExternalAPI(domain, room_name, width, height, parentNode,
-    configOverwrite, interfaceConfigOverwrite, noSsl) {
+    configOverwrite, interfaceConfigOverwrite, noSsl, jwt) {
```

Comment:
@saghul @emcho @yanas 
What is jwt token for ?
I added it as parameter but anyone can still join to my conference via link ?
So whats the point of jwt parameter ?